src/qml/controls/IconTabButton.qml

```diff
@@ -63,7 +58,7 @@ TabButton {
             horizontalAlignment: Text.AlignHCenter
             font.pointSize: 8
             text: control.text
-            color: control.isCurrentTab ? control.theme.textColorSelected : control.theme.textColor
+            color: control.isCurrentTab ? control.palette.highlight : control.palette.windowText
```

Comment:
Shouldn’t this be `highlightedText`? (and elsewhere)

Reply:
No, in this case the text should be orange like the icon.